packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4825,19 +4962,17 @@ export default class Meeting extends StatelessWebexPlugin {
 
         return Promise.reject(error);
       })
-      .then((join) => {
+      .then(async (join) => {
```

Comment:
Would adding awaits in these functions cause any problems in other operations on the meeting?